js/src/gtag-events/utils.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Internal dependencies
+ */
+import { sendToGroup } from './constants';
+
+/* global glaGtagData */
+
+/**
+ * Track an event using the global gtag function.
+ *
+ * @param {string} eventName
+ * @param {Object} eventParams
+ */
+export const trackEvent = ( eventName, eventParams ) => {
+	if ( typeof gtag !== 'function' ) {
+		throw new Error( 'Function gtag not implemented.' );
+	}
+	// eslint-disable-next-line no-console
+	console.log( `Tracking event ${ eventName }` );
```

Comment:
❓ 
Do we want to show a log with the event? As far as I know there are tools to inspect this without logs

Reply:
It's code copied from the WooCommerce Blocks extension when it tracks events, but we can remove it.